rpc_jobs/rpc_ceph.yml

```diff
@@ -207,3 +207,24 @@
     jobs:
       - 'Pull-Request-Whisperer_{repo}'
 
+- project:
```

Comment:
I believe this should already be addressed by "rpc-ceph-re-release-pr" which is defined above. It creates the job https://rpc.jenkins.cit.rackspace.net/job/RE-Release-PR_rpc-ceph-master/ which includes testing the release notes part of the release process. Is that not sufficient?

Reply:
@git-harry I did not add this in.   So Does it need to be removed?     If so I can remove it.   Right now I'm trying to figure out why the Jira unit test failed

Reply:
Why would you want to remove "rpc-ceph-re-release-pr"? What is the need for this pull request if "rpc-ceph-re-release-pr" already exists?

The unit test failure looked transient to me, it should go away with a recheck.